src/Controller/NetsEasyController.php

```diff
@@ -52,12 +52,14 @@ public static function create(ContainerInterface $container) {
   public function createPayment(Request $request) {
     // @todo Implement error handling.
     $amountToPay = floatval($request->get('amountToPay'));
-    $is_test_mode = $this->paymentHelper->getTestMode();
-    $endpoint = $is_test_mode ? 'https://test.api.dibspayment.eu/v1/payments' : 'https://api.dibspayment.eu/v1/payments';
+    $callbackUrl = $request->get('callbackUrl') ?? NULL;
```

Comment:
You don't need ` ?? NULL` here. `$request->get(…)` return `NULL` if the key does not exist.

Does it make sense to use `NULL` as callback URL? It not, you should throw an error if it's not sent to the script.